### PR TITLE
Added submenu nr 5

### PR DESCRIPTION
**Pull request by VonErnest**
Added missing getssets to Order, implemented Repository order, added submenu NR 5 and method to list all orders

Couldn't test it, but added also comments to my method in submenu class.

### CODE REVIEW

File: practical_project/src/main/java/model/Customer.java [hunk 1]
```diff
@@ -48,5 +48,4 @@ public void setPhoneNumber(String phoneNumber) {
                 ", phoneNumber='" + phoneNumber + '\'' +
                 '}';
     }
-
```
Comment (UserOfSda) [inline practical_project/src/main/java/model/Customer.java:51]: Unnecessary

File: practical_project/src/main/java/persistence/RepositoryOrder.java [hunk 2]
```diff
@@ -1,4 +1,20 @@
...
 public class RepositoryOrder {
+
+    private EntityManager em;
+
+    public RepositoryOrder() {
+        em = DBUtil.getEntityManager();
+    }
+
+    public List<Order> listAllOrders() {
+        return em.createQuery("SELECT orderId, orderDate, shippedDate, customerId ORDER BY shippedDate").getResultList();
```
Comment (UserOfSda) [inline practical_project/src/main/java/persistence/RepositoryOrder.java:18]: must also include a table where you select from

**Review by UserOfSda — COMMENTED** [review]
I leaved two comments that i think you must change

File: practical_project/src/main/java/persistence/RepositoryOrder.java (hunk 2 above)
Comment (VonErnest) [reply practical_project/src/main/java/persistence/RepositoryOrder.java:18]: thanks

**Review by UgoHenry — APPROVED** [review]

**Pull request closed without merging**